### DISCUSSION TITLE
3.15版本怎么隐藏文件夹

### DISCUSSION THREAD

**Discussion by daihaosh** [AlistGo/alist#3985] [Q&A] [answered]
隐藏根目录下的“隐藏测试”文件夹，表达式该怎么写？我查到的几种写法都试了。“隐藏测试”，“^隐藏测试$"，”/隐藏测试/“,”隐藏测试.hide“，”hide:隐藏测试“，这几种写法都不行。系统是win2016，alist版本3.15。

**Comment by Jeamorg** [comment, marked as answer]
隐藏教程

将在根目录下的 test 和 example 文件夹隐藏

在元信息里面添加：

- 途径填
    /
- 隐藏填
    test
    example

此时 test 和 example 文件夹已经被隐藏起来了，但是没有密码只能登录后访问（Admin 是可以看见隐藏的文件夹/文件的）

为 test 文件夹添加密码：

在元信息里面添加：

- 途径填
    /test
- 密码填
    123

此时 test 文件夹可以通过密码访问了

![图片](https://user-images.githubusercontent.com/93057377/228411066-9c32e1a8-c6f1-4ec2-badd-bac701eb3373.png)

**Reply by daihaosh** [reply]
![批注 2023-03-29 181142](https://user-images.githubusercontent.com/49355425/228502372-20922f5f-4753-418f-b94b-0cb646184588.jpg)
![批注 2023-03-29 181015](https://user-images.githubusercontent.com/49355425/228502380-44668a0b-a5c0-46f0-94d6-3c21806c8c03.jpg)
试过了不行。文件列表是非管理员用户的，还是显示出来了

**Reply by daihaosh** [reply]
是对的，我权限设置错了
